src/app/(admin)/admin/_lib/validationSchemas/specialistSchema.js

```diff
@@ -59,6 +60,17 @@ const restEditProps = zSpecialistSchema.extend({
     .array()
     .default([])
     .refine(singlePrimaryAddressRefine, { message: MESSAGES.singlePrimaryAddress }),
+  therapiesIds: zStringArray,
+  therapyPrices: z.array(
+    z.object({
+      id: z.string(),
+      price: z.number().nullish(),
```

Comment:
Why do we need `.nullish()` here and allow `null` or `undefined` for price?

Reply:
Good catch, it's not needed here. Removed